### PR TITLE
DB/Graveyards: Added a comment field to describe the graveyards in `g…

### PR DESCRIPTION
**Pull request by Aokromes**
…raveyard_zone`

**Changes proposed**:
- Reduce db diff between branches

**Target branch(es)**: 335

f2527f1f5a86563f214c6d086b2b465644c43107

### CODE REVIEW

File: sql/base/dev/world_database.sql [hunk 1]
```diff
@@ -992,17 +992,18 @@ CREATE TABLE `game_event_seasonal_questrelation` (
 /*!40101 SET character_set_client = @saved_cs_client */;
```
Comment (DDuarte) [inline sql/base/dev/world_database.sql]: Changes to this file should be reverted. world_database.sql only gets updated on TDB releases
(Travis starts by creating the world db, applies this file and only then all the SQL updates)

File: sql/updates/world/3.3.5/2016_05_24_00_world_335.sql [hunk 2]
```diff
@@ -0,0 +1,682 @@
...
+UPDATE `graveyard_zone` SET `Comment`='Westfall, Sentinel Hill GY - Westfall' WHERE `ID`=4;
+UPDATE `graveyard_zone` SET `Comment`='Loch Modan, Thelsamar - Loch Modan' WHERE `ID`=6;
+UPDATE `graveyard_zone` SET `Comment`='Wetlands, Crossroads GY - Dun Morogh' WHERE `ID`=7;
+UPDATE `graveyard_zone` SET `Comment`='Wetlands, Crossroads GY - Wetlands' WHERE `ID`=7;
+UPDATE `graveyard_zone` SET `Comment`='Wetlands, Crossroads GY - Loch Modan' WHERE `ID`=7;
+UPDATE `graveyard_zone` SET `Comment`='Badlands, Graveyard NE - Badlands' WHERE `ID`=8;
+UPDATE `graveyard_zone` SET `Comment`='Badlands, Graveyard NE - Loch Modan' WHERE `ID`=8;
+UPDATE `graveyard_zone` SET `Comment`='The Barrens, The Crossroads - Northern Barrens' WHERE `ID`=10;
+UPDATE `graveyard_zone` SET `Comment`='Desolace, Ethel Rethor GY - Desolace' WHERE `ID`=31;
+UPDATE `graveyard_zone` SET `Comment`='Desolace, Ethel Rethor GY - Maraudon' WHERE `ID`=31;
```
Comment (ForesterDev) [inline sql/updates/world/3.3.5/2016_05_24_00_world_335.sql]: You replace comment above. Should use in query GhostZone:

UPDATE `graveyard_zone` SET `Comment`='Desolace, Ethel Rethor GY - Desolace' WHERE `ID`=31 AND `GhostZone` = 405;
UPDATE `graveyard_zone` SET `Comment`='Desolace, Ethel Rethor GY - Maraudon' WHERE `ID`=31 AND `GhostZone` = 2100;

**Comment by ForesterDev** [comment]
https://gist.github.com/ForesterDev/a85d3ee43c7e2d0c06a1c0c8b7e3cc15

**Comment by ForesterDev** [comment]
Please, look at my gist above. My queries make comments more informative

**Comment by Aokromes** [comment]
I don't see the point of add those "AND `GhostZone`" when it's unique entry.

**Comment by lyn1337** [comment]
The tool used by ForesterDev probably added these, since ID and GhostZone are PRIMARY keys, along with the apostrophes around the numbers. Also, its 'ID' now, not 'Id', I think.

**Comment by ForesterDev** [comment]
Edited.

And as lyn1337 said:

PRIMARY KEY (`ID`,`GhostZone`)

**Comment by lyn1337** [comment]
@ForesterDev Aokromes is still right, there is no point in putting those, it could be misleading…
And single quotes around numeric values are not recommended. Though, ofc, it works. (:

**Pull request merged**